packages/rsk3-utils/src/index.js

```diff
@@ -137,10 +137,12 @@ const isBN = (object) => {
 const KECCAK256_NULL_S = '0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470';
 
 const keccak256 = (value) => {
+    if (!isString(value)) {
+        throw new Error('number parameter should be a number or string.');
```

Comment:
You are checking !iString(), where's the check for a number?

'number parameter should be a number or string.' What's "number" parameter?